install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -45,9 +45,9 @@ spec:
             name: serving-cert
             readOnly: true
         env:
-          - name: KUBERNETES_SERVICE_PORT # allows CVO to communicate with apiserver directly on same host.
+          - name: KUBERNETES_SERVICE_PORT # allows CVO to communicate with apiserver directly on same host.  Is substituted with port from infrastructures.status.apiServerInternalURL if available.
```

Comment:
`... Is substituted with port from infrastructures.status.apiServerInternalURL if available, defaulting to 443 if apiServerInternalURL is set but defines no port`?  Or somehow working the 443 default in here...

Reply:
Never mind, I'll handle this where `apiServerInternalURL` is defined in openshift/api.

Reply:
> Never mind, I'll handle this where apiServerInternalURL is defined in openshift/api.

openshift/api#704.  Marking this thread resolved.